tslib.es6.js

```diff
@@ -107,14 +107,20 @@ export function __exportStar(m, exports) {
 }
 
 export function __values(o) {
-    var m = typeof Symbol === "function" && o[Symbol.iterator], i = 0;
+    var m = typeof Symbol === "function" && o[Symbol.iterator];
     if (m) return m.call(o);
-    return {
-        next: function () {
-            if (o && i >= o.length) o = void 0;
-            return { value: o && o[i++], done: !o };
-        }
-    };
+    if (typeof o === "object") {

```

Comment:
This would break `for..of` on strings for `--target ES5 --downlevelIteration`.

Reply:
Thanks, forgot about array likes / strings, (as below)